### PR TITLE
change_logger_config

### PR DESCRIPTION
**Pull request by borovHokage**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/655

### CODE REVIEW

File: src/UserService/Program.cs [hunk 1]
```diff
@@ -12,16 +12,26 @@ public static void Main(string[] args)
         {
             var configuration = new ConfigurationBuilder()
                 .AddJsonFile("appsettings.json")
-#if DEBUG
-                .AddJsonFile("appsettings.Development.json")
```
Comment (BelehovEgor) [inline src/UserService/Program.cs:16]: Why did you deleted it? We have not any info in those files?

File: src/UserService/Program.cs [hunk 2]
```diff
@@ -12,16 +12,26 @@ public static void Main(string[] args)
...
+            {
+                seqApiKey = configuration["Serilog:WriteTo:1:Args:apiKey"];
+            }
+
             Log.Logger = new LoggerConfiguration().ReadFrom
                 .Configuration(configuration)
                 .Enrich.WithProperty("Service", "UserService")
+                .WriteTo.Seq(
+                    serverUrl: seqServerUrl,
+                    apiKey: seqApiKey)
```
Comment (BelehovEgor) [inline src/UserService/Program.cs:34]: Why we have 2 arguments, but in appsettings 3

**Review by BelehovEgor — CHANGES_REQUESTED** [review]

File: src/UserService/Program.cs (hunk 2 above)
Comment (borovHokage) [reply src/UserService/Program.cs:34]: Spartak said that the 3rd argument is's nessesary

File: src/UserService/Program.cs (hunk 1 above)
Comment (borovHokage) [reply src/UserService/Program.cs:16]: Spartak said that  "Program.cs" needed only fol logger, so this changes won't affect on Production and Develop appsetings

**Review by MariyaSin — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Pull request merged**